db/migrate/20231116085259_create_relationships.rb

```diff
@@ -0,0 +1,13 @@
+class CreateRelationships < ActiveRecord::Migration[7.0]
+  def change
+    create_table :relationships do |t|
+      t.integer :follower_id, null: false
+      t.integer :followed_id, null: false
+
+      t.timestamps
+    end
+    add_index :relationships, :follower_id
+    add_index :relationships, :followed_id
+    add_index :relationships, [:follower_id, :followed_id], unique: true
```

Comment:
`add_index :relationships, :follower_id` は ユニークインデックスと働きが一緒になるので、不要です。`remove_index` で消しておきましょう。

```suggestion
    add_index :relationships, :followed_id
    add_index :relationships, [:follower_id, :followed_id], unique: true
```